src/zjs_ipm.h

```diff
@@ -95,6 +124,15 @@ typedef struct zjs_ipm_message {
             uint8_t color_b;
             void *buffer;
         } glcd;
+#endif
+
+#ifdef BUILD_MODULE_SENSOR
+        struct sensor_data {
```

Comment:
For external sensors (ie DHT22 etc) we would probably need something like "controller" here so that I can do

new Hygrometer( { controller: "dht22" } ) which would hook up to the right Zephyr driver